mono_repo/lib/src/ci_shared.dart

```diff
@@ -45,11 +45,21 @@ class CIJobEntry {
 
   CIJobEntry(this.job, this.commands);
 
-  String jobName(List<String> packages) {
-    final pkgLabel = packages.length == 1 ? 'PKG' : 'PKGS';
+  String jobName(
+    List<String> packages, {
+    @required bool oneOs,
```

Comment:
Rename these to `includeOs` etc - that is more descriptive and won't make the travis one feel super weird where you pass `true` unconditionally.

Reply:
fixed